tox.ini

```diff
@@ -17,7 +17,7 @@ description =
     py{35,36,37,38,39}: Run unit tests against {envname}.
     du{12,13,14}: Run unit tests with the given version of docutils.
 deps =
-    git+https://github.com/html5lib/html5lib-python  # refs: https://github.com/html5lib/html5lib-python/issues/419
+    git+https://github.com/html5lib/html5lib-python# refs: https://github.com/html5lib/html5lib-python/issues/419
```

Comment:
Looks like a stray change.

Reply:
No it was not. In my environment (Ubuntu WSL on Windows 10) tox interpreted the whole string up to the `#` as URL. This change (or deleting the comment completely) was the only way I could run tox.
I admit I did not research further to find alternative solutions or the real root cause.

Reply:
Ah, ok. Perhaps @tk0miya knows more about the tox stuff?

Reply:
I can't understand what happened in @jbab's environment. But, I feel this change is strange. I wrote it as a comment, not a part of URL. -1 for this change.

Reply:
Right, it's a bit odd. @jbab, if you change this part back and submit a separate issue/PR to discuss this, then we can get the C++ part merged. Thanks.

Reply:
ok, done.